rest-api-spec/src/main/resources/rest-api-spec/api/ml.preview_datafeed.json

```diff
@@ -14,8 +14,7 @@
         {
           "path":"/_ml/datafeeds/{datafeed_id}/_preview",
           "methods":[
-            "GET",
-            "POST"
+            "GET"
```

Comment:
OTOH, I can see the `POST` routes being listed in the server action definition:
```
    @Override
    public List<Route> routes() {
        return List.of(
            new Route(GET, BASE_PATH + "datafeeds/{" + DatafeedConfig.ID + "}/_preview"),
            new Route(GET, BASE_PATH + "datafeeds/_preview"),
            new Route(POST, BASE_PATH + "datafeeds/{" + DatafeedConfig.ID + "}/_preview"),
            new Route(POST, BASE_PATH + "datafeeds/_preview")
        );
    }
```
which aligns with the `.json` spec.

So maybe there is another reason this `POST` request failed?

Reply:
Hmm. That's odd then. Our client tests are running the preview_datafeed.yml, executed through the .NET client. By default, if POST is available, it defaults to that. This is the error we got when it ran:

```
02:08:38 Unsuccessful (405) low level call on POST: /_ml/datafeeds/preview-datafeed-feed/_preview
02:08:38 # Audit trail of this API call:
02:08:38  - [1] BadResponse: Node: https://instance:9200/ Took: 00:00:00.0011756
02:08:38 # OriginalException: Elasticsearch.Net.ElasticsearchClientException: Request failed to execute. Call: Status code 405 from: POST /_ml/datafeeds/preview-datafeed-feed/_preview. ServerError: Type:  Reason: "Incorrect HTTP method for uri [/_ml/datafeeds/preview-datafeed-feed/_preview] and method [POST], allowed: [GET]"
02:08:38 # Request:
02:08:38 <Request stream not captured or already read to completion by serializer. Set DisableDirectStreaming() on ConnectionSettings to force it to be set on the response.>
02:08:38 # Response:
02:08:38 {"error":"Incorrect HTTP method for uri [/_ml/datafeeds/preview-datafeed-feed/_preview] and method [POST], allowed: [GET]","status":405}
02:08:38 
02:08:38 
02:08:38   - preview_datafeed.yml - Test preview aggregation datafeed with doc_count
02:08:38     - Failed: Assert operation NumericAssert Length $body "Long" Reason: Expected 3.000000 = 2.000000 
```

Reply:
My apologies. I see now that the test was running against the 7.12.0-SNAPSHOT in error. This is an issue on my end after all so I'll close this PR!